layouts/partials/header/components/mobile-menu.html

```
@@ -0,0 +1,157 @@
+<div class="flex items-center h-14 gap-4">
+  {{ if .Site.Params.enableSearch | default false }}
+    <button
+      id="search-button-mobile"
+      aria-label="Search"
+      class="flex items-center justify-center bf-icon-color-hover"
+      title="{{ i18n "search.open_button_title" }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)